guides/common/modules/proc_calling-the-api-in-python.adoc

```diff
@@ -152,10 +152,10 @@ except ImportError:
     print("Please install the python-requests module.")
     sys.exit(-1)
 
-SAT = "{foreman-example-com}"
+{project-allcaps} = "{foreman-example-com}"
 # URL for the API to your deployed {ProjectX} server
-{project-allcaps}_API = f"https://\{SAT}/api/"
-KATELLO_API = f"https://\{SAT}/katello/api/v2/"
+{project-allcaps}_API = f"https://{{project-allcaps}}/api/"
```

Comment:
If you already mention `KATELLO`, you might as well make it `FOREMAN_API`.

Reply:
Yes, applied.